packages/devtools-reps/src/reps/function.js

```diff
@@ -74,11 +74,22 @@ function getTitle(grip, props) {
 }
 
 function getFunctionName(grip, props) {
-  let name = grip.userDisplayName
-    || grip.displayName
-    || grip.name
-    || props.functionName
-    || "";
+  let { functionName } = props;
+  let end = functionName.length - 1;
+
+  props.functionName =
+    functionName.indexOf('"') == 0 && functionName.lastIndexOf('"') == end
```

Comment:
I think we could use `functionName.startsWith('"') && functionName.endsWith('"')`.
Also, what should we do if `functionName` is wrapped in single quotes (e.g. `'myName'`) ?
And we should handle cases where we don't have `functionName` as a prop (I expect tests to be broken with this change).

More globally, I wonder if this shouldn't be fixed on the consumer side (here I suppose the debugger): `functionName` is a props that we pass to the Rep "on purpose", as opposed to grip's properties that are defined by the protocol.
In my opinion, it makes more sense to do the `functionName` cleanup there, but I may be wrong. What do you think about it ?

Reply:
I don't know where functionName becomes ""$"". I tried to find it to fix the issue there but couldn't. For single quotes, could we check with 'startsWith()' and 'endsWith()' ?